regression/regression_param.sh

```diff
@@ -315,7 +315,7 @@ elif [[ "$machine" = "Gaea" ]]; then
    export MPI_BUFS_PER_PROC=256
    export MPI_BUFS_PER_HOST=256
    export MPI_GROUP_MAX=256
-   export APRUN="srun --export=ALL --mpi=pmi2 -n \$ntasks"
+   export APRUN="srun --export=ALL -n \$ntasks"
```

Comment:
While `sub_gaea` submits the job with `cpus-per-task=$threads"` and `export OMP_NUM_THREADS=$threads`, the srun command does not include `--cpus-per-task=\$threads`.  The srun commands for Orion, Hercules, and Jet include this string.  The srun command for Hera does not include threads.   Should the srun command for Gaea include `--cpus-per-task=\$threads`?

This is a question, not a request for a change.  What's the correct thing to do here?

Reply:
@RussTreadon-NOAA Since the original GSI sub_gaea file contained `--export=ALL` and `-n $ntasks`, I put those back into srun when trying to debug.  export=ALL is default so should be okay to remove.  And since srun will be using all the resources provided by sbatch, we should be able to remove the ntasks as well. I just kicked off the suite of tests with just srun to see.

Reply:
Thanks for the explanation.   Developers may look at log files for failed ctests, only see `srun --export=ALL gsi.x` and not understand that the submission script specified tasks and threads.

@DavidHuber-NOAA , I vaguely recall a discussion about GSI ctest job configurations in a Hercules or Hera related PR.  Does this ring a bell?

Reply:
@RussTreadon-NOAA Yes, we had to make some memory and `exclusive` changes for Hercules.  Part of that discussion included where flags should be placed -- either at sbatch or srun.  As @DavidBurrows-NCO stated, any flag sent to `sbatch` will be inherited by `srun` unless a different value is sent to `srun`.  I think for most machines, the GSI `sub_` scripts are defining `cpus-per-task` at `sbatch` and `srun` to the same value, but this isn't necessary.